exercise/primzahlen.py

```diff
@@ -0,0 +1,7 @@
+import numbers
+
+for index in range(2,100):
+     if (range(2,100)%index) ==0:
```

Comment:
@gaeschpu 
Prinzipiell gute Überlegung, aber ein `range` kann leider nicht Modulo gerechnet werden.

Lösung: Eine zweite `for` Schleife erstellen und dann den `index2` mit `index` vergleichen.